### DISCUSSION TITLE
Set Minimum Validator Commission to 5%

### DISCUSSION THREAD

**Discussion by Atari-buzzk1LL** [fetchai/FIPs#10] [Ideas]
Currently, there are validators on the Fetch-ai Network that operate at low-to-zero percent commission. This has, unfortunately, led to further centralization of the Fetch-ai Network, not even to a benefit of the validators or the network at large.

To start with, let’s look into how low-to-zero percent commission has been identified as a centralization factor. Currently, only 26% of the total 60 active validators are running low-to-zero percent commission, yet this small set has 40% of all FET staked to them, and make half of the top 10 validators. This skew in the data shows that commission very obviously plays a large factor in how delegators are making their choice of which validator they stake with. This is even more evident when taking into consideration that nearly every low-to-zero percent commission validator has reached its current position with no foundation support. This is worrisome because it means that delegators are likely not looking into outside factors, such as a validator's contributions to the chain, involvement in discussion, uptime, whether they’ve been slashed before, etc, and are merely maximizing APY by staking without a cut, or just a minimal cut, being taken.

As for how this is not beneficial to the validators or the network itself, let's look at incentives. Validators require incentives to run, this is because running a node is not free. Even if you already own your equipment beforehand, you still have to pay for the electricity that runs the equipment. So what happens when a validator is receiving no incentive by running at a low-to-zero percent commission? What incentive is there for them to be proactive on the network and even contribute to its ecosystem or upgrade their equipment? Validators should be paid for their work because their work secures the network. It’s not sustainable long-term to run a validator node at a loss, and as someone who works for the network team, that greatly concerns me as we have to look out for the long-term success of the network.

Another reason why delegators may be seeking low-to-zero percent commission validators is that many new delegators do not quite understand how commission works, especially when they’re new to the idea of staking in the first place. When new delegators see that commission means the validator takes a cut of “their” rewards, they will likely take the route where the validator gets almost nothing. There’s also been a misunderstanding that I’ve seen firsthand with new community members, where some believed that if the validator commission is 5% and the current staking APY is 11%, then they’d only receive 6% staking APY, which leads these users to low-to-zero percent commission validators. The reality is that the commission comes from the total rewards and not the APY. So if the current APY is 10% and a validator has 5% commission, the amount they take from the APY is actually 0.5%, leaving the delegator with 9.5% APY to themselves. More concretely, if someone stakes 1000 FET for a year with 10% APY to a validator with 5% commission, there would be a total reward of 100 FET, where 95 FET goes to the delegator and 5 FETs to the validator.

Other networks in the IBC ecosystem have had discussions/proposals around this, such [Osmosis](https://www.mintscan.io/osmosis/proposals/21), [Juno](https://www.mintscan.io/juno/proposals/3) Network, and even a coin such as [Chihuahua Network](https://www.mintscan.io/chihuahua/proposals/1), all of which have better decentralization than the Fetch-ai Network at this time, and although not voted on, the Cosmos EVM chain, [Evmos](https://medium.com/evmos/the-evmos-token-model-edc07014978b), has also mandated a 5% minimum commission rate at launch with a comparably successful decentralization. The Fetch-ai Network has also had a [discussion](https://docs.google.com/document/d/1NevmJR7yTrZuXPEFJX2ZKvwahtjIos3y0Nw55HARAkc/edit?usp=sharing) around a 5% minimum commission rate in November of 2021 where roughly half of the validators at the time agreed to the 5% commission rate, so there is reason to believe this would still be favourable and we’d now like to take action.

All in all, the environment of low-to-zero percent commission raises concerns for the Fetch-ai Network’s security long term and creates a toxic validator environment where validators feel the need to take losses in order to compete, which is an issue. So we believe that setting a minimum commission rate of 5% is beneficial as it then requires delegators to look into more than just the commission a validator takes, and it also means that validators will have a harder time running their node at a loss. So this helps with the decentralization and the longevity of the network.

**Comment by travellingsoldier85** [comment]
Agree with the proposal 5%. Kooltek68 already set it back to 5%

**Comment by ivandiazperez** [comment]
Well, not much to add from my side. BloClick's been at 5% since long ago :)

**Comment by jurekpl** [comment]
No apps, no dex, nothing on fetch network but you want cut income for stakers. Good job!!!!!!!!

**Reply by juguelio** [reply]
Mobix, resonate, dabbaflow and starfleit on the works. I think there's a buzz on Fetch's ecosystem.

**Reply by Atari-buzzk1LL** [reply]
As @juguelio has stated, the idea that there are no apps built on the network isn't true, and there are actually more on the way. 

Even aside from that, I think you didn't read the section where I outlined what commission is meant to be used for and how without it the validators literally are running at a loss which means the network's long-term health is up in the air. Not only that, but 5% commission means that 95% of the rewards still go to you, so it's not a cut of income in the slightest. 

From what we've seen, those with a commission rate set have been more likely to actually contribute to the network in a lot more ways which is beneficial for everyone, we hope you understand. 

This is also a matter of decentralization first and foremost, APY seeking has unfortunately caused problems for the network's decentralization and many networks have decided to take this action, leading to a positive result. So we're giving it a shot.

**Reply by jurekpl** [reply]
Decentralization? Little network now and you think about decentralization? Make apps decentralization isn't your issue.
Losses? Little network and they think about millions from commissions? Make apps, more users, price will grow, at start they will have losses now but later they will make profit. Now their losses is support for network today. If they don't want losses just quit.
Everything is since 1 year on work and never delivered. Ceo running only on interviews and no info about progress. And nearly everything not delivered.
And you think about validator profit on early stages. 10x more price they will have profit? Who cares. Attack staking users instead of growing network. No info, all delayed or non exist and only you think about "decentralization" which is decision to holders, your job is growing ecosystem with name fetch.ai.
Plus it's already decentralized because no one hold 50% or the biggest stake don't hold more than 1%? And you say it's not decentralized? Make your f**** work good don't think about not your issues. Decentralization will come with network growth. Not because you dream about make move on community money.

**Reply by thebookofhealing** [reply]
Let's be clear here... no one is attacking staking users. I am staking and don't feel attacked at all.

To your point about building out the network... decentralization is a big part of that and having a centralized validator set will definitely deter anyone from investing further in the chain. I don't think what you are asking for and this gov proposal are mutually exclusive. Development + network maturation are both key for the growth of Fetch.ai

**Reply by pce05** [reply]
> "Little network now and you think about decentralization? Make apps decentralization isn't your issue"

So you mean they should focus on pumping the ecosystem and only then start worrying about security? Gotcha! Do you want a quick pump? Go to a meme token. Fetch is working on long-term and real use cases, that's why we need to think about security now before adoption comes to the network.

**Comment by toostewechel** [comment]
Agree, 5% is the norm Cosmos ecosystem wide!

**Comment by juguelio** [comment]
I completely agree with the 5% by default. Less than that centralises the stake which creates huge security issues.

**Comment by thebookofhealing** [comment]
Hey all! I think this is a very important proposal and we should 100% move towards a minimum commission of 5%.

I just want to say that we should make sure that the proposal is for a hard-coded upgrade that mandates the minimum. There was a similar signaling proposal before on Sifchain for a 5% min comm, but it wasn't hard coded, and there are still many validators will <5% comms. I figured, this would be hard coded but just wanted to make sure :)

Also, we can fork this upgrade from another chain if needed.

**Comment by Yellowcowstar** [comment]
1) There is no evidence to show that having minimum fees will lead to decentralisation. 

Atom does not have a minimum fee, and they are doing well. In fact, they have gone through the same thought process previously, and did not follow through. 

https://forum.cosmos.network/t/proposal-12-accepted-are-validators-charging-0-commission-harmful-to-the-success-of-the-cosmos-hub/2505/10

2) in the event the proposal were to be put up, validators should vote for abstain as this is a clear conflict of interest.


3) The foundation wallet should also vote abstain as well. The foundation holds a disproportionate amount of voting power, and having this wallet vote is just another form of centralisation.

**Reply by Atari-buzzk1LL** [reply]
To address your points in order.

1. There IS evidence that minimum fees improve decentralization as I outlined the multiple chains that saw success to varying degrees after doing so or have maintained better decentralization from always having them. Plus I literally provided evidence that many delegators will simply APY seek as they have little understanding of how commission works and why it exists, which is seemingly why 0% commission validators have an abnormal amount of stake when compared to the amount of them offering it.

Also, the link you sent about the ATOM discussion is the signalling proposal that PASSED with ATOM delegators agreeing that 0% commission validators are harmful to the network, which was what the proposal was asking - https://www.mintscan.io/cosmos/proposals/12

So if they actually did decide to put it as a protocol vote then this likely would have already been in effect on the chain.

2. In the event that this proposal goes live, validators are free to vote as they wish, as the protocol allows them to hold an opinion and it is the role of delegators to stake with someone who seemingly votes in a similar direction as them if they plan to not vote themselves, but obviously validators do not get the final say as any FET delegator can override the vote weight of their validator using their FET, by simply voting themselves. If a validator wishes to abstain, that is their decision to make to just simply contribute to quorum, but we do not believe that validators HAVE to abstain when this vote has an impact on the chain they contribute to.

3. The foundation wallet hasn't voted on a single proposal since the last upgrade that changed the chain ID, so this is very likely a non-issue, but if the community wishes for the foundation to contribute to quorum on this, then they can.

**Comment by SupportTerminet** [comment]
Agree! 
At this stage, the number of nodes is already very small, decentralization is not high, and the ecosystem is also in its infancy, it needs to accommodate more people who have no money but have the ability and experience to participate in maintaining network security and eco-building. We need to encourage nodes to compete in a more diversified way, rather than a commission of 0.

**Comment by jurekpl** [comment]
Plus stop delaying osmosis github.
https://frontier.osmosis.zone/pool/684
Here is mobix/ust pool before Luna crash.
23 March not delivered mobix to exchanges, mobix or fetch don't even plan give this to exchanges because you never upload it to osmosis for correct display so Q1-q2 mobix bridge was huge joke and lie.
After your team do your work anyone can create pool with correct display. 
Or you decide how decentralization work for fetch instead of doing work catch community money.
It's decentralized and everyone can decide with listing and buying/selling coin or everyone need someone permission to do decentralized things on fetch?

**Reply by Atari-buzzk1LL** [reply]
The MOBIX team is not the Fetch-ai team, MOBIX is owned by Datarella so this doesn't pertain to us at all. Also no, Osmosis will only list a token on the frontend if a pool is funded first with sufficient liquidity, which is once again, entirely up to the MOBIX team and their planning, or if someone else wants to create a pool through a pull request to Osmosis Frontier since the DEX is permissionless. The pool you've shown there was not created by the team. The original pull request to add the MOBX token to Osmosis Frontier was made by another validator, Cros-nest, when we thought that MOBX may be getting added there, but an official pool was not created and submitted to be added. You can even still see the pull request sitting on the Osmosis Frontier GitHub repo, as a draft: https://github.com/osmosis-labs/assetlists/pull/142

Regardless, this has nothing to do with us fighting for the decentralization of the network. The idea that governance proposals can't be worked on until random tasks are completed that have nothing to do with protocol, is odd. We have many different teams whose job it is to do many different things so that tasks can be completed in different areas.

**Comment by packetstracer** [comment]
We support the proposal for a minimum 5%. It's been helpful in many cases before, to help with network decentralization and avoid future risks and damages in this regard.

**Comment by THGITHUB09** [comment]
The fact that a large validator has recently publicly defamed an individual from the Fetch team and then deleted the whole thread when challenged (don't worry, this was captured in full) adds further weight to this proposal needing to move forward.

In my view, true decentralisation needs to be founded on a sustainable, transparent business model and to that effect, I overwhelmingly support this proposal.

Potential bad actor/s hiding behind low fee's such that they have a large % of the network need to be eliminated (note: I said "potential", but based on some recent engagement I have alarm bells going off in my head).

**Comment by ValidarIOS** [comment]
I support this innovation! Of course, this will not help decentralization very much, but at least it will equalize the chances of all validators...

**Comment by mintiayields** [comment]
I support the introduction of the min commission. However, for legal reasons I will have to abstain. Once this discussion and voting protocol are made public (and vote is public no matter waht) we all may face  legal claims from stakers around the world. Can this be decided and introduced by the company that makes the development? Any chances not to do this through a vote?

**Reply by Atari-buzzk1LL** [reply]
All changes made to the chain, especially on a protocol level require a governance proposal and for the community to decide on it through a vote, so no, the Fetch AI company cannot just make a random decision to have this be implemented. The point is to allow the community to decide outcomes, otherwise, we're not exactly acting in a decentralized way.

**Reply by mintiayields** [reply]
How did other networks introduce the minimum setting? Was it by vote, too?

**Reply by thebookofhealing** [reply]
Yes, every other chain did this by vote. There are no real legal concerns around this.

**Comment by mintiayields** [comment]
Off topic: the current APY does not encourage investors, imho.

**Reply by Atari-buzzk1LL** [reply]
After inflation, our APY is more than 1% larger than even the $ATOM APY and they seem to have encouraged investors just fine. There is only a finite amount of $FET that can be rewarded through staking so the only way to increase that APY would be either to have fewer people staking or to increase the inflation rate, which increasing inflation would only serve to drain the staking pool faster.

**Reply by mintiayields** [reply]
I wouldn't compare FET to ATOM at this point, but the finite amount is certainly a strong reason.
